### PR TITLE
application questions, mentors, users facelifts

### PR DESCRIPTION
**Pull request by shaolinhoward**
# Change log
* rework admin menu
* format several edit/list screens

**Note**: will likely have conflicts with https://github.com/carrot-u/carrot-u-admin-app/pull/38 (I didn't see this PR was out and was left unmerged)

# Manual testing
* Tested locally

# Screenshots
image

image

image

image

# Checklist:
- [x] My code follows the style guideline
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Add tests if required

### CODE REVIEW

**Comment by shaolinhoward** [comment]
ok - that's it for me tonight... feel free to review

**Comment by shaolinhoward** [comment]
updated mentor form after merging in #38 and adding current user pre-filled values:
image

File: app/models/user.rb [hunk 1]
```diff
@@ -12,4 +12,13 @@ def gravatar_url
...
   end
+
+  # meh, it's not great but its the best we can do until we separate the name attribute
+  def first_name
+    name.split.first
+  end
+
+  def last_name
+    name.split.last
+  end
```
Comment (muffy) [inline app/models/user.rb:23]: Why do we care about first/last name vs. just name?

File: app/views/mentors/_form.html.erb [hunk 2]
```diff
@@ -18,17 +18,17 @@
...
       <%= form.label :first_name, class: "input-group mb-3" %>
-      <%= form.text_field :first_name, class: "form-control" %>
+      <%= form.text_field :first_name, class: "form-control", value: current_user.first_name %>
     </div>
 
     <div class="form-group">
       <%= form.label :last_name, class: "input-group mb-3" %>
-      <%= form.text_field :last_name, class: "form-control" %>
+      <%= form.text_field :last_name, class: "form-control", value: current_user.last_name %>
     </div>
```
Comment (muffy) [inline app/views/mentors/_form.html.erb:27]: I'm about to put in a PR to set the user's name based on their email. I don't think this should be part of the form, although we could add a "preferred name" field if we want to be super sensitive.

Comment (shaolinhoward) [reply app/views/mentors/_form.html.erb:27]: Hah I think I just paraphrased what you said in the other pr.. agree 💯. Let's follow up later for this

**Pull request merged**